install.sh

```diff
@@ -440,19 +456,40 @@ update_shared_hooks_if_appropriate() {
 #   1 in case a hook fails, 0 otherwise
 #####################################################
 execute_shared_hooks() {
-    for SHARED_ROOT in ~/.githooks/shared/*; do
-        REMOTE_URL=$(cd "$SHARED_ROOT" && git config --get remote.origin.url)
+    # split on comma and newline
+    IFS=",
+    "
+    for SHARED_REPO in $SHARED_REPOS_LIST; do
+  
+        set_shared_root "$SHARED_REPO"
+
+        # Fail if the shared root is not available
+        if [ ! -d "$SHARED_ROOT" ] || [ ! -d "$SHARED_ROOT/.git" ]; then
+            echo "! Failed to execute shared hook: $SHARED_REPO"
+            echo "! Its is not available. Do run:"
+            echo "!   \$ git hooks shared update"
+            return 1
+        fi
+
+        REMOTE_URL=$(cd "$SHARED_ROOT" && git config -f "$SHARED_ROOT/.git/config" --get remote.origin.url)
```

Comment:
I really dont understand why I need here `-f`
Without it and with `--show-origin` shows that in `test-103.sh` it fails, showing the wrong picked config??
The working directory is not empty and points to the right shared repo clone... I have no clue. Can you help me?

Reply:
Solved: GIT_DIR is set during post-checkout. -> Made a note...